### PR TITLE
Add a Qt6 example config

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: bd-nitrogen8m-qt6.yml [hunk 1]
```diff
@@ -66,7 +66,13 @@ local_conf_header:
...
+    PACKAGECONFIG:remove:pn-opencv = "text"
+    # FIXME: original project has rename its 'master' branch to 'main'
+    SRC_URI:pn-tesseract-lang = "git://github.com/tesseract-ocr/tessdata.git;branch=main;protocol=https"
+    # TODO: avoid this: "... no providers found in RDEPENDS:deepview-rt? [file-rdeps]"
+    INSANE_SKIP:append:pn-deepview-rt = "file-rdeps"
 
   qt6-stuff: |
     IMAGE_INSTALL:append = " qtbase-examples"
-    PACKAGECONFIG_pn-qt-base = "examples"
+    PACKAGECONFIG:pn-qt-base = "examples"
```
Comment (rexut) [inline bd-nitrogen8m-qt6.yml]: Please, squash with parent commit.

File: bd-nitrogen8m-qt6.yml [hunk 2]
```diff
@@ -75,4 +75,4 @@ local_conf_header:
 
   qt6-stuff: |
     IMAGE_INSTALL:append = " qtbase-examples"
-    PACKAGECONFIG:pn-qt-base = "examples"
+    PACKAGECONFIG:pn-qtbase = "examples"
```
Comment (rexut) [inline bd-nitrogen8m-qt6.yml]: Please, squash with parent commit.

File: bd-nitrogen8m-qt6.yml [hunk 3]
```diff
@@ -75,4 +75,4 @@ local_conf_header:
 
   qt6-stuff: |
     IMAGE_INSTALL:append = " qtbase-examples"
-    PACKAGECONFIG:pn-qtbase = "examples"
+    PACKAGECONFIG:append:pn-qtbase = " examples"
```
Comment (rexut) [inline bd-nitrogen8m-qt6.yml:78]: Please, squash with parent commit.

**Review by rexut — CHANGES_REQUESTED** [review]
Looks more as a single commit, please fix it. Then we can submit as rebase to main branch.

**Comment by ghost** [comment]
> Looks more as a single commit, please fix it. Then we can submit as rebase to main branch.

That is what I intended to do. Github has the ability to "Squash and commit". But I can do that manually right now, too.

**Comment by ghost** [comment]
> > Looks more as a single commit, please fix it. Then we can submit as rebase to main branch.
> 
> That is what I intended to do. Github has the ability to "Squash and commit". But I can do that manually right now, too.

Done

**Review by rexut — APPROVED** [review]
Many thanks!

**Pull request merged**